rules/network/discovery_potential_network_sweep_detected.toml

```diff
@@ -4,15 +4,15 @@ integration = ["endpoint", "network_traffic"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2023/05/26"
+updated_date = "2023/07/14"
 
 [rule]
```

Comment:
This one doesn't have `auditbeat-*` added as an index. Is this on purpose?

Reply:
Originally this file was not modified in the previous PR, which explains why I did not update it. I added `auditbeat-*` and `filebeat-*` as they should be compatible.

Reference: https://github.com/elastic/detection-rules/pull/2665/files